src/tracer.js

```diff
@@ -70,17 +70,19 @@ export default class Tracer {
             // Normalize the argument so the implementation is always provided
             // an associative array of fields.
             if (arguments.length === 1) {
-                if (typeof nameOrFields === 'string') {
-                    fields = {
-                        operationName : nameOrFields,
-                    };
-                } else {
+                if (typeof nameOrFields === 'object') {
                     fields = nameOrFields;
+                    nameOrFields = fields.operationName;
+                } else {
+                    fields = {};
                 }
-            } else {
-                fields.operationName = nameOrFields;
             }
-            spanImp = this._imp.startSpan(fields);
+            spanImp = this._imp.startSpan({
+                operationName: nameOrFields,
+                parent: fields.parent ? fields.parent._imp : undefined,
```

Comment:
I understand the change to have the implementation's startSpan get a `imp` object rather than the interface object, however it's unclear to me that this is a "fix" rather than just a "change." Does the prior behavior somehow prevent some code from working? (I assume no, since the called code could always reference `parent._imp`.) Is this more consistent with other code in the package?